src/components/Item.vue

```diff
@@ -1,11 +1,11 @@
 <template>
-  <li class="news-item">
+  <span class="news-item">
     <div>
       <a-player :music="{
-        title: item.title.rendered,
+        title: item.title.rendered || ' ',
```

Comment:
You should probably create a separate player component that reads from the Store rather than create a player at each item.

Reply:
I am not sure I follow here. Do you mean the Item.vue itself should read from the too?